types/vegas/index.d.ts

```diff
@@ -0,0 +1,367 @@
+// Type definitions for vegas 2.5
+// Project: http://vegas.jaysalvat.com
+// Definitions by: Piotr Błażejewicz <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/vegas/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me

Reply:
thank you bot!